src/dbinterface.jl

```diff
@@ -0,0 +1,9 @@
+DBInterface.connect(::Type{Connection}, args...; kws...) = Connection(args...; kws...)
+
+DBInterface.prepare(conn::Connection, args...; kws...) = prepare(conn, args...; kws...)
```

Comment:
Is there a reason why this function needs to have separate, identically named bindings with identical method signatures in DBInterface and LibPQ? If a dependency on DBInterface is added anyway, it seems like the current definition of `prepare` could just extend then reexport the binding from DBInterface. Same for `execute`.

Reply:
LibPQ.jl is not currently fully compliant with the interface. The solution presented here avoids bringing it into compliance in favour of providing partial compliance. The future solution to bring it into compliance is either to augment LibPQ.jl's interface to bring it to compliance, or to wrap that interface with a compliant one, and this strategy allows both options.

An example of partial compliance: `prepare` does not have a do-block form. In LibPQ.jl, I think it would make the most sense if that form actually used the function to operate on the statement and then deallocate the statement at the end, rather than the DBInterface method which says the function should be `f()::DBInterface.Connection` (I don't see why that would be particularly useful). Having separate dispatches for these two functions allows both to be implemented in the future.